### PR TITLE
KAS-3489 add search only on urgent publications option

### PR DESCRIPTION
**Pull request by ValenberghsSven**
# :warning: this PR is mutually exclusive with https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1392 :warning: 
Don't merge both

In this PR, Added the option to search on urgent publications only.

Dependent on https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/292

### CODE REVIEW

**Comment by ValenberghsSven** [comment]
Custom jenkins run.
The testdata zip has not been updated yet, so we are still running against the old index.
http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/custom_frontend_backend_combination/251/

**Comment by ValenberghsSven** [comment]
clicked the wrong button and closed by accident....

File: app/routes/search/publication-flows.js [hunk 1]
```diff
@@ -109,6 +118,10 @@ export default class PublicationFlowSearchRoute extends Route {
       filter[':terms:statusId'] = params.publicationStatusIds;
     }
 
+    if (params.urgentOnly) {
+      filter[':terms:urgencyLevelId'] = this.urgencyLevelSpeed.id;
```
Comment (sergiofenoll) [inline app/routes/search/publication-flows.js]: I think the filter here needs to be `:terms:urgencyLevelId.keyword`. Locally, I can't get the search to only return results marked as urgent. After appending `.keyword` to the filter, it does seem to work as expected.

I assume it's related to how Elasticsearch searches string fields. See this conversation in chat for reference: https://chat.semte.ch/channel/mu-semtech?msg=KEHQtfdWyLWEHj7f5

**Review by sergiofenoll — CHANGES_REQUESTED** [review]

File: app/routes/search/publication-flows.js (hunk 1 above)
Comment (ValenberghsSven) [reply app/routes/search/publication-flows.js]: hmmm, I only changed that near the end, may have missed it.
But then we don't have to use `terms`, `filter['urgencyLevelId'] = this.urgencyLevelSpeed.id;` should work?

**Review by sergiofenoll — APPROVED** [review]

**Review by erikap — APPROVED** [review]

**Pull request merged**